decimal64scan.go

```diff
@@ -41,7 +41,6 @@ func (d *Decimal64) Scan(state fmt.ScanState, verb rune) error {
 	if err != nil {
 		return err
 	}
-
 	// Word-number ([Ii]nf|∞|nan|NaN)
 	word, err := tokenString(state, isLetterOrInf)
```

Comment:
can we perhaps do:

```go
word, err := strings.ToLower(tokenString(state, isLetterOrInf))
```

(continued on line 58

Reply:
alternatively - perhaps function which takes a `tokenString` result and parses it into the associated enums (`flNormal`, `flInf`, `flQNaN`, `flSNaN`) so that this pulled out of the `Scan()` function?

Again, not sure if this runs counter to previous style recommendations